modules/claims_api/spec/models/auto_establish_claim_spec.rb

```diff
@@ -17,7 +17,7 @@
   describe 'pending?' do
     context 'no pending records' do
       it 'should be false' do
-        expect(described_class.pending?('123')).to be(false)
+        expect(described_class.pending?('123_789')).to be(false)
```

Comment:
Can you help me understand why this had to change? I don't see any changes to `pending?`. I also don't see any similar changes to test data. What am I missing?

Reply:
@dvogel I just removed this change, looks like I had stale data in my local test database and didnt get around to addressing it before pushing this.